docs/conf.py

```diff
@@ -0,0 +1,61 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+import toml
+
+sys.path.insert(0, os.path.abspath('../qctrl'))
```

Comment:
I think `os.path.abspath('../qctrl')` needs changing; For some reason, on my machine, changing this to `os.path.abspath(..)` works, while in my opinion it should have been `os.path.abspath('../qctrlopencontrols')`.

Reply:
Great pickup.